src/scripts/axis/number.labels.js

```diff
@@ -35,7 +32,12 @@ module.exports = {
     removeDecimals: function(g){
         var decimalTotal = 0;
         g.selectAll('text').each(function(d){
-            decimalTotal += parseFloat(this.textContent.split('.')[1]);
+            var val0 = parseFloat(this.textContent.split('.')[0]);
```

Comment:
took me a minute to comprehend this line, maybe different variable names would make it easier? e.g. val0 -> numerator, val1 -> denominator ? ... So reading: If (numerator == 0 && denominator == 0) then this.textContent = 0 ?